Makefile

```diff
@@ -51,11 +51,11 @@ build: bindata.go
 	$(MAKE) build/deb/$(NAME)_$(VERSION)_amd64.deb
 
 build/docker:
-	chmod +x build/linux/$(NAME) build/darwin/$(NAME)
```

Comment:
Can we instead keep this and make it:

```
	-chmod +x build/linux/$(NAME) build/darwin/$(NAME)
```

This allows it to continue even on failure: https://stackoverflow.com/a/2670143/1515875

Reply:
No, I don't think so. That would potentially break `make build/docker` (without setting `CIRCLECI=true`) as `Dockerfile.env` relies on these files being executable.

Actually, I'd rather suggest moving this `chmod` into `build` step above, directly after `go build`.

Reply:
Why would that potentially break it?

Reply:
Because if chmod fails for whatever reason, such as permissions messed up, `Dockerfile.dev` will bake non-executable binaries into the image.

I agree that is rare case and not a big deal overall, so if you insist of course it absolutely can be done that way. Still, I'm convinced that the way I did it was simpler and better practice at the same time.

Reply:
I want to clarify that non-dev `Dockerfile` does NOT need these binaries at all. Hence there's no point in having that chmod line in `CIRCLECI=true` branch at all.